customize.sh

```diff
@@ -107,7 +107,7 @@ if [ ! -e "$CONF" ]; then
 
 	echo -e "!!! Your Detach.txt file does not exist !!!"
 	echo -e "!!! Copying Detach file to your storage root!!!"
-	cp -af "$TMPDIR/Detach.txt" "$DetachFile"
+	cp -fr "$TMPDIR/Detach.txt" "$DetachFile"
```

Comment:
Why using -r argument?

Reply:
not really needed but it will make sure to overwrite the file if already there